### PR TITLE
Move migrate_3_0 to migrate_5_0

### PR DESCRIPTION
**Pull request by nik9000**
5.0 is the version now.

### CODE REVIEW

File: docs/reference/migration/migrate_5_0.asciidoc [hunk 1]
```diff
@@ -184,14 +184,14 @@ The `host` field has been removed from the cat nodes API as its value
...
-[[breaking_30_parent_child_changes]]
+[[breaking_50_parent_child_changes]]
 === Parent/Child changes
 
 The `children` aggregation, parent child inner hits and `has_child` and `has_parent` queries will not work on indices
 with `_parent` field mapping created before version `2.0.0`. The data of these indices need to be re-indexed into a new index.
 
 The format of the join between parent and child documents have changed with the `2.0.0` release. The old
-format can't read from version `3.0.0` and onwards. The new format allows for a much more efficient and
+format can't read from version `.0` and onwards. The new format allows for a much more efficient and
```
Comment (dadoonet) [inline docs/reference/migration/migrate_5_0.asciidoc]: Is `.0` intentional here?

**Comment by dadoonet** [comment]
Left a comment. Otherwise LGTM.

**Comment by jasontedor** [comment]
This is fine, but you'll need to merge in my latest change since I just made a change to the document in #16858.

**Comment by nik9000** [comment]
> This is fine, but you'll need to merge in my latest change since I just made a change to the document in #16858.

Got it.

**Pull request merged**